tests/Search.test.js

```diff
@@ -48,28 +46,32 @@ describe('Tests for Search', function() {
         userId = 12345678910112;
         key = 'jjjehdnctsjyieoalskcjdhsnahsadndfnusdfsdfsd=';
 
-        executionPath = path.join(__dirname, 'library');
-        if (isWindowsOS) {
-            executionPath = path.join(__dirname, '..', 'library');
-        }
-        userConfigDir = path.join(__dirname, '..');
-
-        searchConfig = require('../js/search/searchConfig.js');
-        const { Search } = require('../js/search/search.js');
-        SearchApi = new Search(userId, key);
-        launchAgent = require('../js/search/utils/search-launchd.js');
-        realTimeIndexPath = path.join(userConfigDir, 'data', 'temp_realtime_index');
-        tempBatchPath = path.join(userConfigDir, 'data', 'temp_batch_indexes');
-        dataFolderPath = path.join(userConfigDir, 'data');
-        if (fs.existsSync(dataFolderPath)) {
-            deleteIndexFolders(dataFolderPath);
-        }
-        done();
+            executionPath = path.join(__dirname, 'library');
+            if (isWindowsOS) {
+                executionPath = path.join(__dirname, '..', 'library');
+            }
+            userConfigDir = path.join(__dirname, '..');
```

Comment:
why can't we use the actual user data path instead of the app root directory? we'd have anyway encrypted the messages right?

Reply:
In `UserData` path there will be the real message index. So thought of using the root dir for UT.